personalize.module

```diff
@@ -345,7 +346,7 @@ function personalize_start_agents() {
   $all_agents = personalize_agent_load_multiple();
   foreach ($all_agents as $agent_name => $agent) {
     $status = personalize_agent_get_status($agent_name);
-    if ($status == PERSONALIZE_STATUS_PAUSED || $status == PERSONALIZE_STATUS_NOT_STARTED) {
+    if ($status == PERSONALIZE_STATUS_SCHEDULED) {
```

Comment:
This function only gets called by cron as far as I can tell which is why I figured it should only look at scheduled agents as they are already implemented at Conductrics.  What am I missing that you're thinking about here?